var/spack/repos/builtin/packages/py-nbclassic/package.py

```diff
@@ -14,11 +14,13 @@ class PyNbclassic(PythonPackage):
 
     license("BSD-3-Clause")
 
+    version("1.1.0", sha256="77b77ba85f9e988f9bad85df345b514e9e64c7f0e822992ab1df4a78ac64fc1e")
     version("1.0.0", sha256="0ae11eb2319455d805596bf320336cda9554b41d99ab9a3c31bf8180bffa30e3")
     version("0.4.8", sha256="c74d8a500f8e058d46b576a41e5bc640711e1032cf7541dde5f73ea49497e283")
     version("0.3.5", sha256="99444dd63103af23c788d9b5172992f12caf8c3098dd5a35c787f0df31490c29")
     version("0.3.1", sha256="f920f8d09849bea7950e1017ff3bd101763a8d68f565a51ce053572e65aa7947")
 
+    depends_on("python@3.7:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("py-jupyter-packaging@0.9:0", when="@0.3.3:", type="build")
```

Comment:
If I'm reading this correctly, `py-jupyter-packaging` can be version 1, from `pyproject.toml`:

```
[build-system]
requires = [
  "jupyter_packaging~=0.9,<2",
  "jupyter_server>=1.17.0",
  "babel"
]
```

Reply:
There appear to be fewer other dependencies per `setup.cfg`:

```
python_requires = >=3.7
install_requires =
    ipython_genutils
    notebook_shim>=0.2.3
    nest-asyncio>=1.5
    ipykernel # bless IPython kernel for now
```

Reply:
`py-jupyter-packaging` version 1 does not exist:
* https://github.com/jupyter/jupyter-packaging

The build dependencies are unchanged since 0.4.8, the version currently available in Spack:

```
[build-system]
requires = ["jupyter_packaging~=0.9,<2", "babel"]
build-backend = "jupyter_packaging.build_api"
```
* https://github.com/jupyter/nbclassic/blob/v0.4.8/pyproject.toml

So since they already work, felt no change was necessary.

As for the stuff in `setup.cfg`, there are already `depends_on()` lines for those, it turns out that many that are listed are 'excessive' in that they are not for `py-nbclassic` itself, but for a sub-dependency. I've tweaked things with a new/updated commit that notes that change for v1.1.